.github/workflows/nightly.yml

```diff
@@ -106,7 +106,7 @@ jobs:
         with:
           use-cross: ${{ matrix.config.cross }}
           command: rustc
-          args: --release --target ${{ matrix.config.target }} -p jormungandr --bin jormungandr -- -C target-cpu=${{ matrix.config.target_cpu }} -C lto
```

Comment:
you can't use this in the root workspace, you need to be in the directory `cd jormungandr && rustc`...

Reply:
```Run actions-rs/cargo@v1
/usr/share/rust/.cargo/bin/cargo rustc --release --target x86_64-unknown-linux-gnu -p jormungandr --bin jormungandr --no-default-features -- -C target-cpu=generic -C lto
error: --no-default-features is not allowed in the root of a virtual workspace```

Reply:
Sorry, fixed in #2186.